src/main/java/org/cloudfoundry/promregator/cache/AutoRefreshingCacheMap.java

```diff
@@ -255,7 +255,7 @@ public void run() {
 			MDC.remove("AutoRefreshingCacheMap");
 		}
 		
-		private void refreshMap() {
+		private synchronized void refreshMap() {
```

Comment:
Why do you want to make it `synchronized` here?
The only one calling it is line 240, and per CacheMap there can only be exactly one thread running.

Reply:
Happy to actually revert this change now ...

Reply:
Reverted